### PR TITLE
add tips for error executing in README

### PR DESCRIPTION
**Pull request by abdfnx**

### CODE REVIEW

**Comment by abdfnx** [comment]
@dlvhdr

File: README.md [hunk 1]
```diff
@@ -16,6 +16,28 @@ Installation requires a minimum version (2.0.0) of the the Github CLI to support
 gh extension install dlvhdr/gh-prs
 ```
 
+> if you get an error when execute `gh prs`, do these steps:
+
```
Comment (dlvhdr) [inline README.md:20]: Looks good!
I would put it as the last section in the readme and add a title called "Troubleshooting"

**Review by dlvhdr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dlvhdr) [reply README.md:20]: Or maybe a section called "Installing Manually"

Comment (abdfnx) [reply README.md:22]: ok, I'll use `Installing Manually`

**Review by dlvhdr — APPROVED** [review]

**Pull request merged**